diff --git a/.wex/command/samples/examples/explain/code_comment/source.py b/.wex/command/samples/examples/explain/code_comment/source.py
index 09a094bf..3c971063 100644
--- a/.wex/command/samples/examples/explain/code_comment/source.py
+++ b/.wex/command/samples/examples/explain/code_comment/source.py
@@ -1,2 +1,2 @@
-def print_hello_world() -> str:
-    return 'Hello World!'
+def print_hello_world() -> str:  # <------------- The "-> str" part defines the type of the returned variable as string
+    return 'Hello World!'  # <------------- Indeed, the output value is a string
